.travis.yml

```diff
@@ -40,6 +40,8 @@ matrix:
       env: TARGET=docs
     - php: '7.3'
       env: TARGET=lint
+    - php: '7.3'
+      env: TARGET=phpstan
```

Comment:
This file is "auto-generated"… what's the plan here exactly?

Reply:
We could add a new key to the `projects.yml` config, which generates a new travis environment

```
block-bundle:
  branches:
    master:
      php: ['7.3']
      tools:
          - phpstan
      versions:
        symfony: ['4.3']
```